src/pymor/models/iosys.py

```diff
@@ -1727,6 +1753,8 @@ def to_matrices(self):
             The |NumPy array| or |SciPy spmatrix| N or `None` (if Cv is a `ZeroOperator`).
         E
             The |NumPy array| or |SciPy spmatrix| E.
+        Q
+            The |NumPy array| or |SciPy spmatrix| Q.
```

Comment:
Need to add " or `None` (if Q is an `IdentityOperator`)", and similar for P, S, N, E.